### ISSUE TITLE
Suggestion: Remove or buff the step assist armor imbuement

### ISSUE THREAD

**Issue by ProfHugo** [Growlith1223/ArsMagica2#77]
With 1.10 adding step-assist on its own it seems pointless to have this anymore. However, reworking it to allowing jumping over 2 blocks seems like a plausible way to make it viable.

**Comment by Growlith1223** [comment]
1.10's step assist isn't necessarily step assist, it's more of an automatic jump option than anything

**Comment by ProfHugo** [comment]
In my opinion, they pretty much functions the same, and if you were to use step assist with auto jump enabled, step assist is practically useless unless your sneaking

**Comment by Divineaspect** [comment]
I disagree on the same function part. But it pretty clearly does devalue step assist.

**Comment by Zieg777** [comment]
I still prefer the step assist over the auto jump. One doesn't drain your hunger.. Also smoother

**Comment by ProfHugo** [comment]
Yea gotta admit that they work very differently, however I do think that it got devalued pretty badly due to auto jump being a thing now in 1.10. Therefor, I think a small buff or rework is appropriate in order to make it more viable compared to mana/burnout reduction
